### ISSUE TITLE
Message searching

### ISSUE THREAD

**Issue by JuliaSoriaSmith** [Markoudstaal/node-red-contrib-discord-advanced#120]
Is there a way to bulk search messages?
E.g. get the last 100 messages, this is also a partial attempt to make a bulkDelete command

**Comment by javis86** [comment]
I think there are several ways. 
The first results on stackoverflows to look up are:

https://stackoverflow.com/questions/56819979/search-a-given-discord-channel-for-all-messages-that-satisfies-the-condition-and
https://stackoverflow.com/questions/72325096/searching-for-a-message-without-knowing-exact-channel-discord-js

**Comment by JuliaSoriaSmith** [comment]
yes but I mean how do I get the messages from an input node? Using the messageManager and info I can get the message object if I know the ID, but in the case of bulk delete, or just generally wanting to see the last 100 messages then I cannot without going through the discordClient node
